lib/drivers/onetouch/oneTouchVerio.js

```diff
@@ -0,0 +1,730 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2017, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import fs from 'fs';
+import { assign, clone, invert } from 'lodash';
+// eslint-disable-next-line
+import drivelist from 'drivelist';
+import sundial from 'sundial';
+import TZOUtil from '../../TimezoneOffsetUtil';
+import annotate from '../../eventAnnotations';
+import crc from '../../crc';
+
+// load usb module only on Linux (OSX and Windows use block device access)
+// eslint-disable-next-line import/no-extraneous-dependencies
+const usb = (process.platform === 'linux') ? require('usb') : null;
+
+const isBrowser = typeof window !== 'undefined';
+// eslint-disable-next-line no-console
+const debug = isBrowser ? require('bows')('OneTouchVerio') : console.log;
+
+const VENDOR_IDENTIFICATION = Buffer.from('LifeScan\x00');
+const USB_SIGNATURE = {
+  RECEIVE: Buffer.from('USBS'),
+  SEND: Buffer.from('USBC'),
+};
+const USB_BULK_BLOCKSIZE = 512;
+const USB_INQUIRY_BLOCKSIZE = 96;
+const USB_FLAGS_READ = 0x80;
+const USB_FLAGS_WRITE = 0x00;
+const STX = 0x02;
+const ETX = 0x03;
+const RESPONSE_OK = 0x06;
+const CBD_OP_CODE = {
+  INQUIRY: 0x12,
+  WRITE_10: 0x2a,
+  READ_10: 0x28,
+};
+const CBD_TRANSFER_LENGTH = 1;
+const LBA_NUMBER = {
+  GENERAL: 3,
+  PARAMETER: 4,
+};
+const BLOCKDEVICE_BLOCKSIZE = 512;
+const BLOCKDEVICE_SIGNATURE = 'LIFESCAN   FAT16';
+const SERVICE_ID = 0x04;
+const QUERY_TYPE = {
+  serialNumber: 0x00,
+  deviceModel: 0x01,
+  softwareVersion: 0x02,
+  unknown: 0x03,
+  dateFormat: 0x04,
+  timeFormat: 0x05,
+  vendorUrl: 0x07,
+  languages: 0x09,
+};
+const QUERY_NAME = invert(QUERY_TYPE);
+const PARAMETER_TYPE = {
+  timeFormat: 0x00,
+  dateFormat: 0x02,
+  displayUnit: 0x04,
+};
+const PARAMETER_NAME = invert(PARAMETER_TYPE);
+const UNIT_OF_MEASURE = [
+  'mg/dL',
+  'mmol/L',
+];
+const TIMESTAMP_EPOCH = 946684800;
```

Comment:
Can we add a comment after this? Something like `// January 1, 2000 12:00:00 AM GMT`.